spec/selenium/conferences/big_blue_button_conference_spec.rb

```diff
@@ -0,0 +1,104 @@
+  include ConferencesCommon
+  include WebMock::API
+
+  bbb_endpoint = 'bbb.instructure.com'
+  bbb_secret = '8cd8ef52e8e101574e400365b55e11a6'
+  bbb_fixtures = {
+    :get_recordings => {
+      'meetingID' => 'instructure_web_conference_3Fn2k10wu0jK7diwJHs2FkDU0oXyX1ErUZCavikc',
+      'checksum' => 'ffa33deaa16e039bcbd84df90ed39196e518c69e'
+    },
+    :delete_recordings => {
+      'recordID' => '0225ccf234655ae60658ccac1e272d48781b491c-1511812307014',
+      'checksum' => 'f9a2b4c9b3e0dd611c1b541eee8bb32309a327ae'
+    }
+  }
+
+  before(:once) do
+    initialize_big_blue_button_conference_plugin bbb_endpoint, bbb_secret
+    course_with_teacher(name: 'Teacher Bob', active_all: true)
+    course_with_ta(name: 'TA Alice', course: @course, active_all: true)
+    4.times do |i|
+      course_with_student(name: "Student_#{i + 1}", course: @course, active_all: true)
+    end
+  end
+
+  before(:each) do
+    user_session(@teacher)
+    get conferences_index_page
+  end
+
+  after { close_extra_windows }
+  context 'when a conference is open' do
+    before(:once) {
+      stub_request(:get, 'bbb.instructure.com/bigbluebutton/api/create')
+    }
+
+    context 'and the conference has no recordings' do
+      before(:once) {
+        stub_request(:get, 'bbb.instructure.com/bigbluebutton/api/getRecordings').
+        with(query: bbb_fixtures[:get_recordings]).
+        to_return(:body => big_blue_button_mock_response('get_recordings', 'none'))
+        @conference = create_big_blue_button_conference(bbb_fixtures[:get_recordings]['meetingID'])
+      }
+
+      it 'should not include list with recordings', priority: '2' do
+        verify_conference_does_not_include_recordings
+      end
+    end
+
+    context 'and the conference has recordings' do
+      before(:once) {
+        stub_request(:get, 'bbb.instructure.com/bigbluebutton/api/getRecordings').
+        with(query: bbb_fixtures[:get_recordings]).
+        to_return(:body => big_blue_button_mock_response('get_recordings', 'two'))
+        @conference = create_big_blue_button_conference(bbb_fixtures[:get_recordings]['meetingID'])
+      }
+
+      it 'should include list with recordings', priority: '2' do
+        verify_conference_includes_recordings
+      end
+    end
+
+    context 'and the conference has one recording and it is deleted' do
+      before(:once) {
+        stub_request(:get, 'bbb.instructure.com/bigbluebutton/api/deleteRecordings').
+        with(query: bbb_fixtures[:delete_recordings]).
+        to_return(:body => big_blue_button_mock_response('delete_recordings'))
+        stub_request(:get, 'bbb.instructure.com/bigbluebutton/api/getRecordings').
+        with(query: bbb_fixtures[:get_recordings]).
+        to_return(:body => big_blue_button_mock_response('get_recordings', 'one'))
+        @conference = create_big_blue_button_conference(bbb_fixtures[:get_recordings]['meetingID'])
+      }
+
+      it 'should remove recording from the list', priority: '2' do
```

Comment:
This test is failing.

```
 BigBlueButton conferences when a conference is open and the conference has one recording and it is deleted should remove recording from the list
  Failure/Error: f('.delete_recording_link', new_conference_list)
  
  Selenium::WebDriver::Error::NoSuchElementError:
    Unable to locate element: :css, ".delete_recording_link"
```

Reply:
Yeah, for this one I needed to expand the recordings before executing the click on delete.
```
Finished in 7.67 seconds (files took 18.31 seconds to load)
3 examples, 0 failures

Randomized with seed 61250
```
Although the error you are pointing at makes me think that the test it is not using the correct assets.

Unable to locate element: :css, ".delete_recording_link"

For what other reason the the element .delete_recording_link would not be found. Would it be a  css required?

Reply:
Hi @jfederico , our continuous integration process will compile assets, so I'm not sure if that's what the issue would be. My best guess is that the link is on the page but hidden, or not on the page at all. If this helps, here is a screenshot of the UI when the failure occurs:
![screen shot 2018-02-12 at 3 40 04 pm](https://user-images.githubusercontent.com/6925979/36121307-60f99272-100b-11e8-8ca3-5b899c8f7221.png)

Reply:
Alright. If we discard the assets, the next thing to look up would be the fixtures. Is it possible that they are not getting there?
If they where, the UI sould look like this:

![image](https://user-images.githubusercontent.com/578359/36122175-6a467ba0-1016-11e8-84ad-19613f7495ff.png)

And expanding the recordings like this:

![image](https://user-images.githubusercontent.com/578359/36122203-7ec003c6-1016-11e8-9143-8fe2442c3413.png)